commercetools-models/src/main/java/io/sphere/sdk/producttypes/commands/updateactions/ChangeAttributeOrderByName.java

```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.producttypes.commands.updateactions;
+
+import io.sphere.sdk.commands.UpdateActionImpl;
+import io.sphere.sdk.producttypes.ProductType;
+
+import java.util.List;
+
+public final class ChangeAttributeOrderByName extends UpdateActionImpl<ProductType> {
```

Comment:
I think it makes  sense here to use the `@HasUpdateAction` to generate the update action, try it out and if you have difficulties get back to me.